### DISCUSSION TITLE
Avatar frames

### DISCUSSION THREAD

**Discussion by ThiagoEloy** [NamelessMC/Nameless#2142] [Ideas]
My suggestion is the possibility of giving avatar frames (frames around the user's avatar) to donators,admins and others. Like giving an green frame for emerald donators, iron, gold etc, 
that would be an great feature especially for vips, so they could be identified easily. (Even thought the VIP group thing already makes it easier, why not even easier?)

**Comment by H4nSolo** [comment]
thats a alternativ to the Rang in the User details ;)
nice Idea
maybe as a addon / modul in v2 ;)
